types/wordpress__blocks/index.d.ts

```diff
@@ -70,6 +88,26 @@ export interface Block<T extends Record<string, any> = {}> {
      * Component to render in the editor.
      */
     readonly edit?: ComponentType<BlockEditProps<T>> | undefined;
+    /**
+     * Block type editor script definition.
+     * It will only be enqueued in the context of the editor.
+     *
+     * @see {@link https://developer.wordpress.org/block-editor/reference-guides/block-api/block-metadata/#editor-script}
+     */
+    readonly editorScript?: string;
+    /**
+     * Block type editor style definition.
+     * It will only be enqueued in the context of the editor.
+     *
+     * @see {@link https://developer.wordpress.org/block-editor/reference-guides/block-api/block-metadata/#editor-style}
+     */
+    readonly editorStyle?: string;
+    /**
+     * It provides structured example data for the block.
+     *
+     * @see {@link https://developer.wordpress.org/block-editor/reference-guides/block-api/block-metadata/#example}
+     */
+    readonly example?: Readonly<Partial<Block> & { innerBlocks: ReadonlyArray<BlockExampleInnerBlock> }>;
```

Comment:
quick update here: I believe that we need to make `innerBlocks` optional here otherwise people will _have_ to provide inner blocks in their examples even if the blocks don't support them.

```ts
… & { innerBlocks?: ReadonlyArray<BlockExampleInnerBlock> }
```

Reply:
Yes, good point. Fixed it 👍